tests/test_grains_minicircle.py

```diff
@@ -45,24 +45,6 @@ def test_gaussian_filter_minicircle(minicircle_grain_gaussian_filter: Grains, pl
     return fig
 
 
-@pytest.mark.mpl_image_compare(baseline_dir="resources/img/")
```

Comment:
If this test is being removed we should also remove the `tests/img/test_z_threshold_minicircle.png` that is associated with it.

```
git rm tests/img/test_z_threshold_minicircle.png
```

Reply:
I've removed this image and that associated with the `test_mask_overlay`.